config/capsule-answers.yaml

```diff
@@ -0,0 +1,5 @@
+---
+certs:
+  generate: false
+  deploy: true
+capsule: true
```

Comment:
I personally don't think the 'capsule' should enforce puppet to be setup by default - if the smart proxy doesn't do this by default neither should we.